### PR TITLE
Added explosion effect

### PR DESCRIPTION
**Pull request by jorlol**
-created explosion prefab
-added prefeb on building destroy

### CODE REVIEW

File: Assets/Scripts/Destroyable.cs [hunk 1]
```diff
@@ -104,4 +105,10 @@ void TrackBuilding()
         Destroy(gameObject);
        // Debug.Log ("added building " + buildingName);
     }
+
+    private void createExplosion()
+    {
+        GameObject explosion = Instantiate(Resources.Load("Prefabs/explosion")) as GameObject;
```
Comment (Mikedhaan) [inline Assets/Scripts/Destroyable.cs]: Don't load resources every instantiate, load it on the start of the script.

File: Assets/Scripts/Explosion.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    public ParticleSystem Main;
+    public ParticleSystem[] Ring;
+    public SparkParticles Sparks;
+    public ParticleSystem Flash;
+    public ParticleSystem[] Debris;
+
+    void Start()
+    {
+        for (int i = 0; i < Ring.Length; i++)
+            Ring[i].Play();
```
Comment (Mikedhaan) [inline Assets/Scripts/Explosion.cs]: Always use brackets

File: Assets/Scripts/Explosion.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        Sparks.Play();
+        Flash.Play();
+        for (int i = 0; i < Debris.Length; i++)
+            Debris[i].Play();
```
Comment (Mikedhaan) [inline Assets/Scripts/Explosion.cs]: Same as last comment - Always use brackets

File: Assets/Scripts/SparkParticles.cs [hunk 4]
```diff
@@ -0,0 +1,44 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class SparkParticles : MonoBehaviour
+{
+    public Transform BillboardTarget;
+    ParticleSystem _particleSystem;
+    ParticleSystem.Particle[] _particles = new ParticleSystem.Particle[25];
```
Comment (Lansenou) [inline Assets/Scripts/SparkParticles.cs]: _underscore????

Comment (jorlol) [inline Assets/Scripts/SparkParticles.cs]: Suggested by Visual Studio.. but fixed.

**Pull request merged**